### PR TITLE
Improve erigon build and add docker-compose for op replica

### PR DESCRIPTION
**Pull request by boyuan-chen**
This PR changes `chain name` and `chain id` in erigon build to environment variables. By doing this, we can use the image for any chains including replicas for Optimism and local devnet.

The `docker-compose` files for the Optimism's replica is added too.

### CODE REVIEW

File: boba-community/jwt-secret.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+688f5d737bad920bdfb2fc2f488d6b6209eebda1dae949a8de91398d932c517a
```
Comment (jyellick) [inline boba-community/jwt-secret.txt]: I'm guessing this wasn't meant to be checked in? Maybe add to the `.gitignore`?

File: boba-community/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  mkdir erigon
+  cd ./erigon
+  tar xvf ~/[DIR]/op-erigon-goerli.tar
+  ```
+
+3. Create a shared secret (JWT token)
+
+  ```bash
+  cd op-erigon
+  openssl rand -hex 32 > jwt.txt
```
Comment (jyellick) [inline boba-community/README.md]: Here we use `jwt.txt` but below, we use `jwt-secret.txt`

File: boba-community/jwt-secret.txt (hunk 1 above)
Comment (boyuan-chen) [reply boba-community/jwt-secret.txt]: Removed. I thought it's easier for them to use a JWT token set by us and set it as the volume in the docker-compose file. For the security consideration, I removed it and ask users to set it up.

File: boba-community/README.md (hunk 2 above)
Comment (boyuan-chen) [reply boba-community/README.md]: Updated

File: boba-community/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  mkdir op-db
+  cd ./op-db
+  tar xvf ~/[DIR]/op-erigon-goerli.tar
+  ```
+
+3. Create a shared secret (JWT token)
+
+  ```bash
+  cd op-anchorage/boba-community
+  openssl rand -hex 32 > jwt-secrete.txt
```
Comment (mmontour1306) [inline boba-community/README.md]: Should be "secret" rather than "secrete"?

Comment (boyuan-chen) [reply boba-community/README.md]: Good catch

**Review by mmontour1306 — APPROVED** [review]

**Review by jyellick — APPROVED** [review]

**Pull request merged**